src/main/webapp/translation.html

```diff
@@ -24,7 +24,6 @@
     </header>
     <footer>
     </footer>
-    <script src="script.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
This script tag should go at the bottom too

Reply:
my b